DSharpPlus.Commands/CommandsExtension.cs

```diff
@@ -35,6 +38,15 @@ namespace DSharpPlus.Commands;
     System.Threading.Tasks.ValueTask<string?>
 >;
 
+using ParameterCheckFunc = System.Func
+<
+    object,
+    DSharpPlus.Commands.ContextChecks.ParameterChecks.ParameterCheckAttribute,
+    DSharpPlus.Commands.ContextChecks.ParameterChecks.ParameterCheckInfo,
+    CommandContext,
+    System.Threading.Tasks.ValueTask<string?>
+>;
```

Comment:
Delegate?
```suggestion
private delegate ValueTask<string?> ParameterCheckFunction(object invoker, ParameterCheckAttribute attribute, ParameterCheckInfo parameter, CommandContext context)
```

Reply:
no, Func<...> is more flexible and more easily understood across different callsites (this includes people who write their own command executors and potentially want to manipulate the delegate). the only advantage it'd provide is specifying the names, which i feel like are fairly self-explanatory here